data/com.endlessm.Metrics.policy

```diff
@@ -38,4 +38,14 @@
     </defaults>
   </action>
 
+  <action id="com.endlessm.Metrics.ResetTrackingId">
```

Comment:
Is this used? I don't see any reference to it. I think you need to update `on_authorize_method_check`.

Reply:
Ah! looks like I overzealously reverted this change. I'll unrevert it, since we should make resetting the tracking ID a privileged operation.